EmailSlots/src/components/Email.jsx

```diff
@@ -0,0 +1,26 @@
+import { Regards } from "./Regards";
+import { Salutation } from "./Salutation";
+import PropTypes from "prop-types"
+
+export const Email = ({
+  receiverName = "Sir",
```

Comment:
Is this the correct place for these default arguments?

Reply:
In JS seems fine to me. could have also set the default value inside the component.
One other way that I could find was setting static property `defaultProps` to Email. But it is deprecated.
Please suggest if there is any other better way.

Reply:
Specifically, what I meant was that the Regards property is intrinsic to the Regards component and should be a default argument on the internal component. If the component is called outside of <Email /> component, it will not have the default argument which is expected.